assets/js/components/Heater.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+	<div class="vehicle pt-4">
```

Comment:
Die Kopie der Vehicle.vue gefällt mir nicht. Das beißt und später bei der Weiterentwicklung. Ich mach mal nen Optimierungsvorschlag auf dieser Basis.

Reply:
Die Temperaturauswahl ist auch nicht schön- wer will schon in 5er-Schritten Warmwasser machen. Da geht sicher noch Einiges. Habs erstmal separat gemacht damit nicht allzuviel if-else in die Experimentierphase kommt. Kannst natürlich alles ändern was Du siehst.